changelog/7342.trivial.rst

```diff
@@ -0,0 +1 @@
+`~sunpy.net.jsoc.attrs.Notify` checks for a valid input value.
```

Comment:
```suggestion
`~sunpy.net.jsoc.attrs.Notify` checks that a valid email address has been given as a value.
```
Does this have any other higher-level user facing behaviour that needs mentioning? e.g. if I forget to do a JSOC Fido search with an email address, does this now raise a more helpful error?